src/middlewares/__tests__/cors.js

```diff
@@ -0,0 +1,77 @@
+const middy = require('../../middy')
+const cors = require('../cors')
+
+describe('📦 Middleware CORS', () => {
+  test('It should default to "*"', () => {
+    const handler = middy((event, context, cb) => {
+      cb(null, {})
+    })
+
+    handler.use(cors())
+
+    const event = {
+      httpMethod: 'GET'
+    }
+
+    handler(event, {}, (_, response) => {
+      expect(response).toEqual({
+        headers: {
+          'Access-Control-Allow-Origin': '*'
+        }
+      })
+    })
+  })
+
+  test('It should not override already declared cors header', () => {
```

Comment:
Here as well might be more explicit something like:

`It should not override already declared Access-Control-Allow-Origin header`